lib/viewcomponents/graphCanvas.jsx

```diff
@@ -0,0 +1,54 @@
+var React = require('react');
+var GraphModel = require('./../model/graph');
+var Graph = require('./graph.jsx');
+
+var GraphCanvas = React.createClass({
+  propType: {
+    rootGraph: React.PropTypes.instanceOf(GraphModel)
+  },
+
+  getInitialState: function() {
+    return {
+      navigationStack: [this.props.rootGraph]
```

Comment:
I suppose that your intention is to push things onto this stack at some point in order to expand a container to the full canvas. I don't see you actually doing that anywhere, though.